pkg/network/tcp_transport.go

```diff
@@ -78,9 +76,7 @@ func (t *TCPTransport) Accept() {
 
 func (t *TCPTransport) isCloseError(err error) bool {
 	if opErr, ok := err.(*net.OpError); ok {
-		if reClosedNetwork.Match([]byte(opErr.Error())) {
-			return true
-		}
+		return errors.Is(opErr, net.ErrClosed)
```

Comment:
What about removing `isCloseError` and just using `errors.Is` at the call-site? Seems quite readable to me.